CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Add: Do not create initial entity when a new device is provisioned (just when appendMode is enabled)
```

Comment:
```suggestion
- Add: do not create initial entity when a new device is provisioned and appendMode is false
```

(If I have understood the functionality correctly...)

Btw, no issue number associated?